### ISSUE TITLE
Matchups command fails with in leagues with category scoring

### ISSUE THREAD

**Issue by DMcP89** [DMcP89/harambot#74]
**Describe the bug**
The matchups command fails to retrieve match ups when running against an NBA fantasy league

**To Reproduce**
1. Install and configure harambot with a yahoo NBA fantasy leauge
2. Add harambot to your discord server
3. run $mstachups
4. See logs

```
2022-10-11T04:56:38.974695+00:00 app[worker.1]: Error while fetching matchups for league: 15944
2022-10-11T04:56:38.974708+00:00 app[worker.1]: Traceback (most recent call last):
2022-10-11T04:56:38.974709+00:00 app[worker.1]:   File "/app/harambot/yahoo_api.py", line 131, in get_matchups
2022-10-11T04:56:38.974710+00:00 app[worker.1]:     team1_projected_points = team1[1]['team_projected_points']['total']
2022-10-11T04:56:38.974710+00:00 app[worker.1]: KeyError: 'team_projected_points'
```

**Expected behavior**
Matchups card returned

**Comment by DMcP89** [comment]
Seems the issue is not with NBA leagues but with the category scoring type. Yahoo's API does not return the team_projected_points field when the league's scoring type is set to categories.

**Comment by easyjbreezy** [comment]
Hey DMc - first time using any bots and our group is loving it to start the year. We are having trouble with $stats "Player" showing "Player not found", $matchups showing "I'm having trouble getting that right now please try again later", and $trade not returning anything (a trade is processing but maybe needs to go through fully in order to show up?)

Similar to the start of the thread, we are a category league so I'm guessing that is why the matchups command isn't working but wanted to report the others as well. Thank you so much for your time making and sharing this bot :)

**Comment by DMcP89** [comment]
@easyjbreezy yes each of these commands has a bug in it when dealing with category scoring. A fix for this is in progress.

**Comment by DMcP89** [comment]
Fixed in #77 Merged to 0.3.0-Beta branch
